nxdrive/engine/engine.py

```diff
@@ -682,6 +684,19 @@ def _create_local_watcher(self) -> LocalWatcher:
     def _get_db_file(self) -> Path:
         return self.manager.home / f"ndrive_{self.uid}.db"
 
+    def _disk_space(self) -> tuple:
+        Disk = namedtuple("Disk", "total free used local_sync_files")
```

Comment:
You should move this at the top of the file and rename it to `DiskUsage`.
Also rename the last field to `synced` to be consistent with one-word (and we will know what it is about).